src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -489,7 +487,58 @@ select game
                 tournamentSсheduleInfo);
         }
 
-        private void ValidateGameInRoundOnCreate(
+        private static void freeDayGameValidation(bool GameValidationIsFreeDayGame, TournamentScheduleDto tournamentScheduleInfo, GameResultDto game)
+        {
+            if (GameValidationIsFreeDayGame)
+            {
+                if (tournamentScheduleInfo.Scheme != TournamentSchemeEnum.PlayOff)
+                {
+                    throw new ArgumentException(
+                    Resources
+                    .SameFreeDayGameInRound);
+                }
+                else
+                {
+                    throw new ArgumentException(
+                        string.Format(Resources.SameTeamInRound, game.HomeTeamName));
+                }
+            }
+            else
+            {
+                throw new ArgumentException(
+                   string.Format(
+                   Resources.SameGameInRound,
+                   game.HomeTeamName,
+                   game.AwayTeamName,
+                   game.Round.ToString()));
+            }
+        }
+
+        private static void theSameTeamInTwoGames(bool GameValidationIsFreeDayGame, TournamentScheduleDto tournamentScheduleInfo, GameResultDto game, Game newGame)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method 'theSameTeamInTwoGames' to match camel case naming rules, consider using 'TheSameTeamInTwoGames'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS100)